operator/pom.xml

```diff
@@ -36,7 +36,6 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <quarkus.operator.sdk.version>6.0.1</quarkus.operator.sdk.version>
-        <quarkus.version>3.0.0.Final</quarkus.version>
```

Comment:
The Operator uses an independent Quarkus version from Keycloak. That's because the Operator uses [QOSDK](https://github.com/quarkiverse/quarkus-operator-sdk) which uses Quarkus as a transitive dependency. Hence, we should stick here to whatever Quarkus version does QOSDK [use](https://github.com/quarkiverse/quarkus-operator-sdk/blob/6.0.1/pom.xml#L15). Eventually, once we start using the LTS versions of Quarkus and QOSDK, they should be the same but that's not the case right now.

Reply:
Hmm. Ok, this should be possible to do, but the operator must use a distinct version property. I will change this PR to implement that